src/aws/Makefile

```diff
@@ -39,9 +45,9 @@ output:
 	docker-compose run --rm app /bin/terraform output
 
 .PHONY: init
-init: ## Initialize Terraform
+init: env.d/development ## Initialize Terraform
```

Comment:
`env.d/state_bucket` is also needed here.

Reply:
state_bucket was useless and has now been removed.